### ISSUE TITLE
Option to remove menubar icon

### ISSUE THREAD

**Issue by vjpr** [devhubapp/devhub#196]
My menubar is full. Even with "menubar mode" turned off it still appears up there. It should not force an icon up there.

**Comment by Godsentp** [comment]
I need help please

**Comment by Godsentp** [comment]
Hello I need help [godsentpaulomi@gmail.com](godsentp@github.com)

**Comment by ashrash** [comment]
I would like to contribute !

**Comment by satyaarthchhabra** [comment]
Hi , can you give me some more idea on the design are you talking about removing this 
![image](https://user-images.githubusercontent.com/56510402/129165243-b2ea9459-bbc6-45e6-a211-9b16d03ef071.png)

**Comment by fuckpass8629** [comment]
ふくい しょうき Sharp
@shaashaa420

**Comment by vidit21srivastava** [comment]
Is this issue still open?
I would like to work on this issue.

**Comment by prithvi009** [comment]
is this issue still open?

**Comment by najmulhc** [comment]
Is this issue solved yet? If not you may allow me to fix it.

**Comment by suyashsuvare** [comment]
Hey, I can see this issue is still in open state. I am trying to fix it.

**Comment by doiayokanmi** [comment]
Can I still contribute to this issue?

**Comment by dmaysx6** [comment]
Yes absolutely please Jesus.  I have short term memory problems and I can't
foe the life of me EVER remember my passwords.  Even though I just made
it.

**Comment by dmaysx6** [comment]
No wrong person lol

**Comment by vikas-pal-ts** [comment]
is anyone working on this feature?
I would like to contribute

**Comment by dmaysx6** [comment]
Feel free

**Comment by theRaven-code** [comment]
this issue hasn't been fixed yet? 
Can I contribute?

**Comment by MannCode** [comment]
Is this issue still open. I want to contribute

**Comment by vikas-pal-ts** [comment]
No, it's not fixed yet. 
@dmaysx6 I didn't see your DM. @theRaven-code @MannCode You guys can work on it if you want, and I will also be free starting next week to start contributing.
see you..

**Comment by codvik** [comment]
Is this issue still open?

**Comment by Hamjaster** [comment]
Is this issue open? I wanna contribute !
